app/feedperson/tests.py

```diff
@@ -12,3 +12,23 @@ def setUpTestData(cls):
   def test_str_value(self):
     feed_person = FeedPerson.objects.get(id=1)
     self.assertEqual(feed_person.name, str(feed_person))
+
+  def test_load_feed_people_removes_original_data(self):
+    feed_person = FeedPerson.objects.get(id=1)
+    load_feed_people()
+    self.assertEqual(FeedPerson.objects.filter(eppn=feed_person.eppn).count(), 0)
+
+  def test_load_feed_people_imports_western_ave_people(self):
+    load_feed_people()
+    western_ave_people = filter(lambda person: "Western Ave" in person.location, FeedPerson.objects.all())
+    non_western_ave_people = filter(lambda person: "Western Ave" not in person.location, FeedPerson.objects.all())
+    self.assertTrue(len(list(western_ave_people)) > 0)
+    self.assertEqual(len(list(non_western_ave_people)), 0)
+
+  def test_load_feed_people_returns_expected_keys(self):
```

Comment:
This test is really just testing that the FeedPeople model has the appropriate fields. If you have just one object, with just name and location, and the rest null, this test will still resolve to true. I don't think this is necessary to test here.

Reply:
Ok, this test was removed in 31bb7f4.